packages/reactivity/src/hooks/raw.ts

```diff
@@ -0,0 +1,27 @@
+import type { Raw } from '@/types'
+
+import { unref } from './ref'
+
+export const toRaw = <T extends object>(target: T): Raw<T> => {
+  const obj = {} as Raw<T>
+
+  const descriptors = Object.entries(target).reduce<
+    PropertyDescriptorMap & ThisType<T>
+  >((acc, [k, v]) => {
+    let result: PropertyDescriptor = {
+      get: () => unref(v),
+      enumerable: true,
+    }
+
+    if (typeof v === 'function') {
+      result = { value: v, enumerable: true }
+    }
```

Comment:
!

Reply:
original result value has a getter and there we assign value, so I'll need to manually delete the get field, it will look awful